edgexfoundry/app-service-configurable
# feat(snap): add additional tokens for app-service-configurable profiles

This PR adds updates the snap so that the secret-store tokens for
1. external-mqtt-trigger
2. push-to-core

are copied from the edgexfoundry snap. Both are built-in profiles that are referenced in the TAF tests

Signed-off-by: Siggi Skulason <siggi.skulason@canonical.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-service-configurable/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions



To test:
1.  This PR depends on the edgexfoundry also being updated in PR https://github.com/edgexfoundry/edgex-go/pull/3825. Use the edgexfoundry snap from that PR. 
2. Build the edgex-app-service-configurable snap
3. Install it and then connect the interfaces so that it gets the token: `sudo snap connect edgexfoundry:edgex-secretstore-token edgex-app-service-configurable`
4. set the profile:`sudo snap set edgex-app-service-configurable profile=external-mqtt-trigger`
5. start the service and confirm in the journal that it has started with the secret-store token


## New Dependency Instructions (If applicable)
